frb_codegen/src/ir/ty_delegate.rs

```diff
@@ -48,6 +52,8 @@ impl IrTypeTrait for IrTypeDelegate {
                 "ZeroCopyBuffer_".to_owned() + &self.get_delegate().dart_api_type()
             }
             IrTypeDelegate::PrimitiveEnum { ir, .. } => ir.safe_ident(),
+            IrTypeDelegate::Backtrace => "String".to_owned(),
+            IrTypeDelegate::Anyhow => "Anyhow".to_owned(),
```

Comment:
uber-nit: Shall we name the dart class `FrbAnyhowException` or something like that, since in the Dart world, people usually call all kinds of exception `___Exception`